topics/libcloud/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: apache-libcloud
```

Comment:
This alias topic has not been applied to any repositories. Can we remove this?

Reply:
Removed in e3f55c9c5fe6c67ba602d94e56884f33e997cb22.